manifests/a/Atominn/WonderPen/2.0.5/Atominn.WonderPen.locale.en-US.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Atominn.WonderPen
+PackageVersion: 2.0.5
+PackageLocale: en-US
+Publisher: oldj
```

Comment:
Is this correct? Other versions have ```Publisher: Atominn```

Reply:
Yes for this version atleast that is correct

Reply:
cc @msftrubengu: Looks like this version has a new publisher in Control Panel:
![image](https://user-images.githubusercontent.com/11600822/148603278-031a9a68-585d-41b5-88da-b6362d495f54.png)
![image](https://user-images.githubusercontent.com/11600822/148603323-3a826c17-5167-4eda-b89e-f87a317fb0a8.png)